packages/tasks/src/model-libraries.ts

```diff
@@ -111,6 +111,16 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 			wildcard: { path: "*.npz" },
 		},
 	},
+	bm25s: {
```

Comment:
LGTM, but let's try to switch to explicitly setting this as a library (`library_name: bm25s`) in the repos rather than using `tags`

Reply:
Agreed, I'll assist with getting that done

Reply:
done!